.github/workflows/ci.yml

```diff
@@ -219,19 +219,18 @@ jobs:
           override: true
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install linker
-        run: sudo apt-get install libssl-dev cmake pkg-config zlib1g-dev
-      - name: Install tarpaulin
+      - name: Install llvm-cov
         uses: actions-rs/install@v0.1
         with:
-          crate: cargo-tarpaulin
+          crate: cargo-llvm-cov
```

Comment:
This needs rust toolchain component to work.

Reply:
Done.